hack/test-cmd.sh

```diff
@@ -78,9 +84,15 @@ set -e
 
 function find_tests {
   find "${OS_ROOT}/test/cmd" -name '*.sh' -not -wholename '*images_tests.sh' | grep -E "${1}" | sort -u
```

Comment:
That I 100% agree with, but do you want me to rewrite `cleanup` so that it can be called basically from the start of the script so you don't get undeclared variable errors?